Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```diff
@@ -32,3 +32,16 @@
     contents:
     - id: BoxSurvivalMedical
     - id: RubberStampPsychologist
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackCaptain
```

Comment:
This should parent off of the default `ClothingBackpack` for the same reason as the other removed Captain items. Also applies to the duffel and satchel